test/openssl/test_ssl.rb

```diff
@@ -1774,7 +1782,7 @@ def test_get_ephemeral_key
 
   def test_fallback_scsv
     supported = check_supported_protocol_versions
-    return unless supported.include?(OpenSSL::SSL::TLS1_1_VERSION) &&
+    omit unless supported.include?(OpenSSL::SSL::TLS1_1_VERSION) &&
       supported.include?(OpenSSL::SSL::TLS1_2_VERSION)
```

Comment:
We can assume all libssl versions support TLS 1.2 and TLS 1.3, so a check for TLS 1.2 is no longer necessary. #841 overlooked this.

I agree an explicit omission would be useful here, since TLS 1.1 is not typically disabled, and this should be noted.

```suggestion
    omit "TLS 1.1 is required to run this test" unless supported.include?(OpenSSL::SSL::TLS1_1_VERSION)
```